.eslintrc.js

```diff
@@ -40,6 +40,12 @@ module.exports = {
             ),
         ],
 
+        // Ban matrix-js-sdk/src imports in favour of matrix-js-sdk/src/matrix imports to prevent unleashing hell.
+        "no-restricted-imports": ["error", {
```

Comment:
if possible, can we also restrict use of `matrix-js-sdk` as a bare import? It leads to all kinds of fun issues in forks & runtime.

Reply:
Sure, frustratingly the patterns option doesn't work for us here so its quite a repetitive rule